frontend/src/components/VPopover/meta/VPopover.stories.mdx

```diff
@@ -1,36 +1,39 @@
-import { placements as popoverPlacements } from "@popperjs/core"
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs"
 
+import { placements as popoverPlacements } from "@popperjs/core"
 import { log } from "~/utils/console"
 
 import VPopover from "~/components/VPopover/VPopover.vue"
 import VButton from "~/components/VButton.vue"
 
-export default {
-  component: VPopover,
-  title: "Components/VPopover",
-  argTypes: {
-    hideOnEsc: "boolean",
-    hideOnClickOutside: "boolean",
-    autoFocusOnShow: "boolean",
-    autoFocusOnHide: "boolean",
-    placement: {
-      type: "radio",
-      options: [...popoverPlacements],
-    },
-    label: "text",
-    labelledBy: "text",
-  },
+<Meta title="Components/VPopover" components={VPopover} argTypes={{}} />
+
+export const popoverArgTypes = {
+  hideOnEsc: { control: { type: "boolean" } },
+  hideOnClickOutside: { control: { type: "boolean" } },
+  autoFocusOnShow: { control: { type: "boolean" } },
+  autoFocusOnHide: { control: { type: "boolean" } },
+  placement: { options: [...popoverPlacements], control: { type: "radio" } },
+  label: { control: { type: "text" } },
+  labelledBy: { control: { type: "text" } },
+  close: { action: "close" },
+  open: { action: "open" },
 }
```

Comment:
Instead of repeating arg types in each story, they can be set for the entire component.

```suggestion
<Meta title="Components/VPopover" components={VPopover} argTypes={{
  hideOnEsc: { control: { type: "boolean" } },
  hideOnClickOutside: { control: { type: "boolean" } },
  autoFocusOnShow: { control: { type: "boolean" } },
  autoFocusOnHide: { control: { type: "boolean" } },
  placement: { options: [...popoverPlacements], control: { type: "radio" } },
  label: { control: { type: "text" } },
  labelledBy: { control: { type: "text" } },
  close: { action: "close" },
  open: { action: "open" },
}} />
```